provisioning/kubernetes/build.sh

```diff
@@ -1,6 +1,21 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
 
+# Etcd cluster for Templates API
+: "${ETCD_REPLICA_COUNT:=3}"
+ETCD_ROOT_PASSWORD_BASE64=$(kubectl get secret --namespace default templates-api-etcd -o jsonpath="{.data.etcd-root-password}" 2>/dev/null || echo -e '')
+if [[ "$ETCD_ROOT_PASSWORD_BASE64" == "" ]]; then
+  # Generate random root password if it not set
+  ETCD_ROOT_PASSWORD_BASE64=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 17 | base64 || echo -e '')
```

Comment:
Jo tak tady mi to failuje. Ale `LC_CTYPE=C` pomůže (https://unix.stackexchange.com/a/45406)
![CleanShot 2022-08-23 at 10 25 25](https://user-images.githubusercontent.com/215660/186110044-64f0aeef-24aa-46ae-ad75-6a7b27a1dee5.jpg)
![image](https://user-images.githubusercontent.com/215660/186110236-3bf6a314-0d00-4ba0-b263-4b399b15be69.png)

Reply:
:+1: